docs/content/users/quickstart.md

```diff
@@ -70,6 +70,34 @@ Please note that you will need to change the PHP version to 7.4 to be able to wo
     ddev launch
     ```
 
+## CiviCRM (Standalone)
+
+[CiviCRM Standalone](https://civicrm.org/blog/ufundo/next-steps-civicrm-standalone) allows running [CiviCRM](https://civicrm.org/) without a CMS.
+
+```bash
+mkdir my-civicrm-site && cd my-civicrm-site
+ddev config --project-type=php
```

Comment:
From https://docs.civicrm.org/installation/en/latest/standalone/

> `public/media/`	Public files uploaded to your site, like images that appear on a contribution page or in a mass email

```suggestion
ddev config --project-type=php --upload-dirs=public/media
```

Reply:
I never had any use for that, and for a Quick Start it's overkill IMHO.

Reply:
Let's add it, to be safe.